### DISCUSSION TITLE
Release 5.1.0.1 ML NF testing Status for Sunbird ED

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Ed/Community#358] [General Discussion]
Hi All,

Please find the testing status of R5.1.0.1 as of 18th Jan 2023.

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (41.05% Completed)
Planned Sign-Off Date: 20th Jan 2023

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/213188323-573964a2-a6a1-4f17-8ba2-66d65fe2f11c.png)

![image](https://user-images.githubusercontent.com/40622169/213188429-fc6e8b31-89e5-44de-b9ee-cf5d547d7074.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/213188593-c7ec723b-fcfb-470b-abff-e4f9b984bc94.png)

CC @vinukumar-vs  @rajeevsathish @Krishnaj20  @swayangjit @chitranshu-keshav @ShailandraOjha @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.1.0.1 as of 19th Jan 2023.

Environment: Sunbird ED Staging
Testing completion percentage: In Progress (98.68% Completed)
Planned Sign-Off Date: 20th Jan 2023

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/213469028-059e9a6e-92cd-421c-9da1-07380768fa4b.png)

![image](https://user-images.githubusercontent.com/40622169/213469163-28efca40-9cb4-4022-aac5-d0c7566311db.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/213469431-5f659b25-d671-4ad1-880f-41b9de9b8f78.png)

CC @vinukumar-vs @rajeevsathish @Krishnaj20 @swayangjit @chitranshu-keshav @ShailandraOjha @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.1.0.1 as of 20th Jan 2023.

Environment: Sunbird ED Staging
Testing completion percentage: Delayed (99.47% Completed)
Planned Sign-Off Date: 20th Jan 2023

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/213724948-4b09dea6-c33a-47e0-af8a-5c5efa95ce2b.png)

![image](https://user-images.githubusercontent.com/40622169/213725069-8c6219a6-db95-450b-996e-83719978b20f.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/213725195-63a024c8-a3bf-4d4b-be30-81f9086acd26.png)

CC @vinukumar-vs @rajeevsathish @Krishnaj20 @swayangjit @chitranshu-keshav @ShailandraOjha @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.1.0.1 as of 23rd Jan 2023.

Environment: Sunbird ED Staging
Testing completion percentage: Delayed (99.74% Completed)
Planned Sign-Off Date: 20th Jan 2023

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/214058311-e5953ce7-337b-443f-8897-f13131bb6830.png)

![image](https://user-images.githubusercontent.com/40622169/214058371-6b0c90f3-e08f-4840-be26-74bafa299974.png)

Defect Status:
![image](https://user-images.githubusercontent.com/40622169/214058459-42156a44-aaa6-40b8-97fe-9ed8e911ebfc.png)

CC @vinukumar-vs @rajeevsathish @Krishnaj20 @swayangjit @chitranshu-keshav @ShailandraOjha @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094 @ashokreddy1208

**Comment by luckynvdu** [comment]
@Krishnaj20 @vinukumar-vs When are we planning to close the existing testing on 5.1.0.1 so that the branch is created for 5.1.3.0 code merge and testing for coKreat?

@mmanimegalai @nadiya496 @anunagaraj

**Comment by anunagaraj** [comment]
Hi All,

Please find the testing status of R5.1.0.1 as of 24th Jan 2023.
- We have completed the Execution and are pending with only **1 open defect (ED-1136) as its a P1**  and this defect will be converted as a Bug as per the comment ED-1134

Environment: Sunbird ED Staging
Testing completion percentage: Delayed (100% Completed)
Planned Sign-Off Date: 20th Jan 2023

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
![image](https://user-images.githubusercontent.com/40622169/214312660-d3f9ded5-eb29-4ca4-8dc3-86bfd07a9e4f.png)

![image](https://user-images.githubusercontent.com/40622169/214313484-82cdf048-4ee0-4abb-982d-0cc875a9a1c9.png)

Defect status:
![image](https://user-images.githubusercontent.com/40622169/214313084-4925bc13-28e9-4f3a-9ae5-ce8c6b898372.png)

CC @vinukumar-vs @rajeevsathish @Krishnaj20 @swayangjit @chitranshu-keshav @ShailandraOjha @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094 @ashokreddy1208 @mmanimegalai

**Comment by ShailandraOjha** [comment]
Hi All,

Please find the testing status of R5.1.0.1 as of 25th Jan 2023.

Note: We are still left with 1 open issue ED-1136 and waiting for the build to verify. Unless the issue is fixed we won't be able to provide the sign-off.

Environment: Sunbird ED Staging
Testing completion percentage: Delayed (100% Completed)
Planned Sign-Off Date: 20th Jan 2023
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1EqJTBOmDjl3JYWQF3FCnT1YyC29MJwyMTJg-rHn0-2A/edit#gid=1581973510)

Execution Status:
<html>
<body>
<google-sheets-html-origin>

% Pending Overall | % Executed Overall
-- | --
0.0 | 100.00


</body>
</html>

Defect status:

![image](https://user-images.githubusercontent.com/44188853/214593380-274c191f-8989-4e75-af76-091cb6a495d7.png)

CC @vinukumar-vs @rajeevsathish @Krishnaj20 @swayangjit @chitranshu-keshav @vijiurs @Preethi-Sl @anisha147 @Aparna-karri591 @Nadiya496 @Naveenkumar739 @PoornimaRajesh @Priyakashi @manju4201 @AdarshP29@Ak0094 @ashokreddy1208 @mmanimegalai
@anunagaraj 

Regards,
Shailandra Ojha
